tests/test_backend.py

```diff
@@ -324,8 +334,14 @@ def setUp(self):
     def tearDown(self):
         # Empty outbox everytime test finishes
         FakeSESConnection.outbox = []
-
+    
+    def test_from_email(self):
+        settings.AWS_SES_FROM_EMAIL = "return@example.com"
+        send_mail('subject', 'body', 'from@example.com', ['to@example.com'])
+        self.assertEqual(self.outbox.pop()['Source'], 'return@example.com')
```

Comment:
And this is the correct behaviour

```suggestion
        settings.AWS_SES_FROM_EMAIL = "my_default_from@example.com"
        send_mail('subject', 'body', 'ignored_from@example.com', ['to@example.com'])
        self.assertEqual(self.outbox.pop()['Source'], 'my_default_from@example.com')
```

Reply:
Thanks, so this is just an example if I understand, the test is fine before this suggestion.

Reply:
Yes. The behavior of the test is correct already.